doc: extensions: initial version of manifest_revisions_table (#6838)
nrfconnect/sdk-nrf: doc/nrf/introduction.rst

See :ref:`getting_started` for information about how to install the |NCS| and about the first steps.
See :ref:`dev-model` for more information about the |NCS| code base and how to manage it.

> .. _project_revisions:

Reviewer: ```suggestion
.. _repository_revisions:
```

> .. _repository_revisions:

Reviewer 2: Please remove this change, already a part of https://github.com/nrfconnect/sdk-nrf/pull/6848

Author: done

> Repository revisions
> ********************
> 

Reviewer: This sentence is still missing:
```suggestion
Repository revisions
********************

The following table lists the different repositories and revisions that are included as part of |NCS| |release| release:
```

Author: done

Repositories and revisions
**************************

> The following table lists the different repositories and revisions that are included as part of |NCS| |release| release:

Reviewer 2: ```suggestion
The following table lists all the repositories (and their respective revisions) that are included as part of |NCS| |release| release:
```

Author: done

Repositories and revisions
**************************

> The following table lists all the repositories (and their respective revisions) that are included as part of |NCS| |release| release:

Reviewer: Oh! Looks like this change did not get added to my commit.@gmarull could you fix this?
```suggestion
The following table lists all the repositories (and their respective revisions) that are included as part of |NCS| |version| release:
```
